drivers/bmx280/include/bmx280_internals.h

```diff
@@ -113,7 +113,7 @@ extern "C" {
  * @{
  */
 #define CALIB_T_P_BASE                  (BMX280_DIG_T1_LSB_REG)
-#define CALIB_T_P_LEN                   (17U)
+#define CALIB_T_P_LEN                   (32U) /* might be larger than neeeded */
```

Comment:
previous value was "17". The highest accessed index that popped up int the warning was "23". I chose 32 as the next highest power of two, on stack it would probably have chosen "24".

Reply:
> The 8-bit registers are named calib00...calib41 and are stored at memory addresses 0x88...0xA1 and 0xE1...0xE7.

so 26 should do

Reply:
If I'm correct this should be 0x17 or 0d22. But we can change that in a follow-up with some more thorough testing. 32U should definitely be enough.

Reply:
Reading the data sheets, it should be ~~26~~ 24 (last byte is reserved) for both [bmp280](https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp280-ds001.pdf) and [bme280](https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bme280-ds002.pdf)

Reply:
Since those are turned into register reads, I'd rather not read additional random registers without knowing if there might be some effect on-read.

Reply:
Ok, let's merge as is then I open a PR for the correct value?

Reply:
Why not use the right value right away?

Reply:
> Why not use the right value right away?

Because it will block all PRs for another 20 minutes :|

Reply:
> Ok, let's merge as is then I open a PR for the correct value?

Yes please!